src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -522,7 +521,7 @@ else if (ast.getType() == TokenTypes.METHOD_DEF) {
             result = ignoreOverriddenMethods && hasOverrideAnnotation(modifiers);
         }
         else {
-            result = CheckUtil.isReceiverParameter(ast);
+            result = ast.findFirstToken(TokenTypes.IDENT) == null;
```

Comment:
`ast.findFirstToken(TokenTypes.This_literal) != null`

Reply:
https://github.com/checkstyle/checkstyle/pull/13436#discussion_r1292108696

Reply:
we return to the util method